redeem/gcodes/M2x.py

```diff
@@ -249,15 +251,28 @@ def execute(self, g):
             return
 
         fn = text.strip()
+	list_location = None
 
         if fn.startswith('/usb'):
             fn = fn.replace('/usb', USB_MOUNT_LOCATION)
+	    list_location = USB_MOUNT_LOCATION
```

Comment:
Mixed tabs and spaces - looks like the existing code uses spaces but your editor wants tabs?